pom.xml

```diff
@@ -22,38 +22,37 @@
   <description> A statistics framework used inside Ehcache and the Terracotta products</description>
   <url>https://github.com/Terracotta-OSS/terracotta-management/</url>
 
+  <prerequisites>
```

Comment:
I don't think other terracotta-oss are using the toolchain no? offheap-store doesn't. And that can be another PR :-)